src/Kunstmaan/PagePartBundle/Tests/Form/HeaderPagePartAdminTypeTest.php

```diff
@@ -6,20 +6,13 @@
 use Kunstmaan\PagePartBundle\Tests\unit\Form\PagePartAdminTypeTestCase;
```

Comment:
This use causes the failing tests (maybe we should search all use statements for the old /unit/ namespace)

Reply:
Thanks, will do 👍🏻

Reply:
Done 👍  Lets see if the tests pass!